src/generator/operations.ts

```diff
@@ -726,6 +731,11 @@ function createInterfaceDefinition(group: OperationGroup, imports: ImportManager
     }
     const returns = generateReturnsInfo(op, false);
     interfaceText += `\t${opName}(${getAPIParametersSig(op, imports)}) (${returns.join(', ')})\n`;
+    // Add resume LRO poller method for each Begin poller method
+    if (isLROOperation(op) && !op.extensions!['x-ms-pageable']) {
+      interfaceText += `\t// Resume${pascalCase(op.language.go!.pollerType.name)} - Used to create a new poller from a resume token of a previously implemented poller\n`;
```

Comment:
"Used to create a new instance of this poller from the resume token of a previous instance of this poller type."  Something like that maybe.

Reply:
Done